test/e2e/antctl_test.go

```diff
@@ -290,22 +290,29 @@ func TestAntctlProxy(t *testing.T) {
 		return nil
 	}
 
+	addressV4 := "127.0.0.1"
+	addressV6 := "::1"
```

Comment:
what if the Node on which the proxy runs only supports IPv4 or IPv6?

Reply:
This line of code handles the logic:
https://github.com/vmware-tanzu/antrea/pull/1823/files#diff-e1915c091016e9984424957bddd2b15f841fbfdd4f12ebaccc082c9c2cdb0a0bR309
Is this what you are asking?

Reply:
yes thanks, but I think you should use `t.Skipf("Cluster network family requirements not me")` instead of a simple `return` statement

Reply:
Good idea. Updated.